delete.php

```diff
@@ -56,4 +56,3 @@
 } else {
   header("Location: index.php?sid=my-doggos");
 }
```

Comment:
Tento komentář jsem vůbec nepochopil :smiley: - `// Kontrola, zda je uživatel povolený`

Nevím zda ukládání chybových hlášek do SESSION je good practise.. asi spíš ne...

A ohledně těch zanořených IFů: je to strašně nečitelné, o dost lepší je využívat buďto tzv. `Guard Clauses` https://betterprogramming.pub/refactoring-guard-clauses-2ceeaa1a9da

nebo funkcionalitu ifů zabalovat do funkcí/tříd :+1: 

pomohli mi např. toto: https://www.youtube.com/watch?v=g2nMKzhkvxw

Reply:
// Kontrola, zda je uživatel povolený - To znamená, jestli je uživatel oprávněný upravovat psa (Je jeho vlastníkem), v tomto případě odstraňovat :) špatně jsem to interpretovala, budu vědět na příště

Děkuji také moc za odkazy!